docs/ci.md

```diff
@@ -0,0 +1,39 @@
+# Continuous Integration
+
+This documents describes the CI process for the solution - in partucular for the Spin apps.
+
+## Integration Tests
+
+The workflow [PR](../.github/workflows/PR.yaml) is used as a validation pipeline for pull requests created against the main branch.
+
+The job contains the generation and setup of the devcontainer including the docker-in-docker installation. That is required to build the wasm apps because it can be configured to use the containerd runtime for pulling and storing images as explaining in the [Docker-In-Docker](../docs/ci.md#docker-in-docker) section.
```

Comment:
`use the containerd runtime` -> `use the wasm based containerd runtime`

Reply:
done